ci/test-conformance-gke.sh

```diff
@@ -95,6 +97,14 @@ case $key in
     K8S_VERSION="$2"
     shift 2
     ;;
+    --antrea-dir)
+    GIT_CHECKOUT_DIR="$2"
+    shift 2
+    ;;
+    --gcloud-dir)
```

Comment:
this is not a directory, you provide the path to exec directly

but I actually suggest using the `GCLOUD` environment variable directly (set it from Jenkins) rather than adding a command-line argument for this

Reply:
Changed the option name to gcloud-path. For Jenkins environment, I tried adding this as a Jenkins env variable as well as exporting it in the vm that actually runs this test, it seems that the env var cannot be picked up if the build step is running the ci/test-conformance-gke.sh rather than using complete build steps. So I'm still keeping the command line argument here.